modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-test/src/testIntegration/java/com/liferay/headless/commerce/delivery/catalog/resource/v1_0/test/ProductResourceTest.java

```diff
@@ -145,6 +171,115 @@ private Product _addCPDefinition(Product product) throws Exception {
 		};
 	}
 
+	private Product _addCPDefinitionWithCustomFields(
+			String customFieldName, String customFieldValue, Product product)
```

Comment:
remove the product parameter and use the random string where you need this (like in the description, name...)